util/admin/admin_integration_test.go

```diff
@@ -82,23 +82,33 @@ func (s *testAdminSuite) TestAdminCheckTable(c *C) {
 		c double          as (JSON_EXTRACT(k,'$.c')),
 		d decimal(20,10)  as (JSON_EXTRACT(k,'$.d')),
 		e char(10)        as (JSON_EXTRACT(k,'$.e')),
-		f date            as (JSON_EXTRACT(k,'$.f')),
```

Comment:
why this is affected?

Reply:
There is a bug with json_extract, previous code contained other bug(type convertion) which covers this bug. After simplify the logic, the latter bug is fixed, so the first bug appear.